{
    "original_source": {
        "database(url)": "https://www.cve.org",
        "software_system(CPE)": [
            "cpe:2.3:a:debian:python-imaging:-:*:*:*:*:*:*:*"
        ],
        "fix_commit": "https://github.com/python-pillow/Pillow/commit/205e056f8f9b06ed7b925cf8aa0874bc4aaf8a7d",
        "version_end_excluding": {
            "debian:python-imaging": "unknown"
        },
        "path": {
            "file_path": ""
        }
    },
    "programming_language": "PYTHON",
    "licences": {
        "database": "",
        "affected_repository": "Other"
    },
    "related_vulnerability_entry": {
        "CVE": [
            "CVE-2014-3589"
        ],
        "CWE": []
    }
}